renovate.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi",
+    "group:reactMonorepo",
+    "group:reactrouterMonorepo"
```

Comment:
By the way, `config:base` includes `group:monorepos` and so you shouldn't have a need to include any `group:<x>Monorepo` presets manually

Reply:
Thanks, I missed that in the documentation.